package.json

```diff
@@ -16,7 +16,7 @@
     "jshint": "~2",
     "postcss-cli": "~2",
     "remark-cli": "~2",
-    "remark-lint": ">=5.1.0",
+    "remark-lint": "~5.1.0",
```

Comment:
Is this change related?
Is the problem that 5.4.0 ([the latest](https://www.npmjs.com/package/remark-lint)) was being installed, and that version is broken??
If yes ... that needs to be clearly stated in the commit message.
If these tests work with 5.4.0, then changing this file should be a separate PR, as it is not related.

(Note that due to caching, it isnt obvious which versions are installed in the logs, unless the build is without the cache)